### PR TITLE
Geometries in search

### PR DESCRIPTION
**Pull request by kalbermattenm**
Actually, it is simpler to use the extent. Otherwise we should have some kind of a specific "treatment" for each type of geometry...

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,112 +1,199 @@
...
-    <meta name="author" content="">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
-    <title>Géoportail LiDAR du SITN</title>
+<head>
+  <meta charset="utf-8">
+  <meta name="description" content="">
+  <meta name="author" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+  <link rel="icon" href="favicon.ico" />
+  <title>Potree Viewer</title>
```
Comment (maltaesousa) [inline index.html]:
```suggestion
  <title>Géoportail LiDAR du SITN</title>
```
`<title>` is often used in SEO, would like to keep LiDAR and SITN together there :)

File: index.html [hunk 2]
```diff
@@ -1,112 +1,199 @@
...
-	<link rel="stylesheet" href="potree/libs/openlayers3/ol.css" type="text/css">
-	<link rel="stylesheet" href="potree/libs/spectrum/spectrum.css" type="text/css">
-  </head>
-
-  <body>
+<body>
+  <script>
+    var isIE = /*@cc_on!@*/false || !!document.documentMode;
+      if (isIE) {
+        alert('Internet Explorer n\'est pas supporté pour cette application, vous allez être redirigés vers le Géoportail 2D');
```
Comment (maltaesousa) [inline index.html:22]: C'est ballot, le géoportail 2D il supporte pas non plus IE :)

File: sitnapp/js/gmf_search_box.js [hunk 3]
```diff
@@ -15,13 +13,11 @@
         },
         source: function( request, response ) {
             $.ajax( {
-                url: "https://sitn.ne.ch/production/wsgi/search?",
```
Comment (maltaesousa) [inline sitnapp/js/gmf_search_box.js:18]: nice

**Review by maltaesousa — APPROVED** [review]
Thanks!!

File: index.html (hunk 1 above)
Comment (kalbermattenm) [reply index.html]: Sorry I missed this one!

File: sitnapp/js/gmf_search_box.js [hunk 4]
```diff
@@ -41,13 +38,26 @@
...
+                // Create an ol feature
+                const geojson = new ol.format.GeoJSON();
+                const geom = geojson.readFeature(ui.item.feature).getGeometry();
+                const extent = geom.getExtent();
+                const center_x = extent[0] + ((extent[2] - extent[0]) / 2);
+                const center_y = extent[1] + ((extent[3] - extent[1]) / 2);
+                coord = [
+                    center_x,
+                    center_y,
+                ]
```
Comment (kalbermattenm) [inline sitnapp/js/gmf_search_box.js]:
```suggestion
                ];
```

**Pull request merged**